docker/.env

```diff
@@ -240,10 +240,15 @@ MF_CASSANDRA_READER_SERVER_KEY=
 
 ### InfluxDB
 MF_INFLUXDB_PORT=8086
-MF_INFLUXDB_DB=mainflux
+MF_INFLUXDB_HOST=mainflux-influxdb
 MF_INFLUXDB_ADMIN_USER=mainflux
 MF_INFLUXDB_ADMIN_PASSWORD=mainflux
-MF_INFLUXDB_HTTP_AUTH_ENABLED=true
+MF_INFLUXDB_ADMIN_URL="http://localhost:8086"
```

Comment:
is `MF_INFLUXDB_ADMIN_URL` used anywhere? If not remove it.

Reply:
No, we define the port 8086 separately, so I will remote this